draft-ietf-dnsop-svcb-https.md

```diff
@@ -508,13 +508,16 @@ procedure:
    acceptable parameters.  This TargetName and SvcParams represent the
    preferred endpoint.  If connection to this endpoint fails, the
    client MAY try to connect using values from a lower-priority record.
-   If all attempts fail, clients SHOULD go to step 5.
+   If all attempts fail, clients SHOULD go to step 5 (except as noted in
+   {{ech-client-behavior}}.
 
-5. Try to use the endpoint consisting of $ADDR_QNAME, the authority endpoint's
+5. There are no usable ServiceMode records.  Clients SHALL connect to the
```

Comment:
```suggestion
5. At this point there are no usable ServiceMode records, either because $SVCB_QNAME has no SVCB records or because no SVCB records have acceptable parameters. In this case, clients SHALL connect to the
```

Reply:
OK, I incorporated a version of this change.